### ISSUE TITLE
RqSocket getLocalAddress and getRemoteAddress can throw UnknownHostException

### ISSUE THREAD

**Issue by kjuby8709gsome** [yegor256/takes#516]
BkBasic puts headers X-Takes-LocalAddress and X-Takes-RemoteAddress with slashes '/':
```
X-Takes-LocalAddress: /127.0.0.1
X-Takes-LocalPort: 8080
X-Takes-RemoteAddress: /127.0.0.1
X-Takes-RemotePort: 43727
```
Thats why RqSocket getLocalAddress and getRemoteAddress throws an exception:
```
java.net.UnknownHostException: /127.0.0.1: unknown error
  at java.net.Inet6AddressImpl.lookupAllHostAddr(Native Method)
  at java.net.InetAddress$2.lookupAllHostAddr(InetAddress.java:928)
  at java.net.InetAddress.getAddressesFromNameService(InetAddress.java:1323)
  at java.net.InetAddress.getAllByName0(InetAddress.java:1276)
  at java.net.InetAddress.getAllByName(InetAddress.java:1192)
  at java.net.InetAddress.getAllByName(InetAddress.java:1126)
  at java.net.InetAddress.getByName(InetAddress.java:1076)
  at org.takes.rq.RqSocket.getLocalAddress(RqSocket.java:57)
```

environment:
java version "1.8.0_66"
Java(TM) SE Runtime Environment (build 1.8.0_66-b17)
Java HotSpot(TM) Server VM (build 25.66-b17, mixed mode)

**Comment by yegor256** [comment]
@kjuby8709gsome thanks for reporting, we'll try to fix soon!

**Comment by davvd** [comment]
@kjuby8709gsome I am aware of the task, give me some time to find a developer...

**Comment by davvd** [comment]
@kjuby8709gsome I added milestone `1.0` to this issue, let me know if there has to be something else

**Comment by davvd** [comment]
@alexey-krylov can you please help? Keep in mind [this](http://www.xdsd.org/2014/04/17/how-xdsd-is-different.html). If you have any technical questions, don't hesitate to ask right here; The budget of this issue is **30 mins**, which is exactly how much will be paid when the task is *done* (see [this](http://www.xdsd.org/2014/04/17/how-xdsd-is-different.html) for explanation)

**Comment by alexey-krylov** [comment]
@davvd I'm still in progress of XDSD learning, but i'm ready to fix this.
I think problem inside BkBasic, slash should be removed.

**Comment by alexey-krylov** [comment]
@davvd, quality analyzers bans my commit. I will setup IDE some time later.

**Comment by dmzaytsev** [comment]
@alexey-krylov I hope it will help

>To avoid frustration, before sending us your pull request please run full Maven build:
    $ mvn clean install -Pqulice

https://github.com/yegor256/takes#how-to-contribute

**Comment by alexey-krylov** [comment]
@dmzaytsev, thanks.

**Comment by davvd** [comment]
> @davvd I'm still in progress of XDSD learning, but i'm ready to fix this.
> I think problem inside BkBasic, slash should be removed.

@alexey-krylov you have to address your technical questions to the ticket author or project architect (@yegor256)

**Comment by alexey-krylov** [comment]
@davvd how can i close this issue?

**Comment by alexey-krylov** [comment]
@kjuby8709gsome issue is fixed. Closing?

**Comment by kjuby8709gsome** [comment]
> @kjuby8709gsome issue is fixed. Closing?

@alexey-krylov yep, thanks!

**Comment by alexey-krylov** [comment]
@davvd issue is closed. Check pls.

**Comment by davvd** [comment]
@kjuby8709gsome these 2 puzzles were created in this ticket: `516-c8aa6c8f`/#533, `516-ffc97ad1`

**Comment by davvd** [comment]
@elenavolokhova please, review this task for compliance with our [quality rules](http://at.teamed.io/qa.html)

**Comment by elenavolokhova** [comment]
@davvd Looks good!

**Comment by davvd** [comment]
> @davvd Looks good!

@elenavolokhova thank you ;)

**Comment by davvd** [comment]
@alexey-krylov I added 10 mins to @elenavolokhova (for QA review) in transaction 75287902; Thanks so much! Your account was topped up for **30 mins** (transaction ID is `AP-6G0026428N427082P`, the task took 72 hours and 8 mins); added +30 to your rating, now it is equal to [+0](http://www.netbout.com/b/36337?open=rating)

**Comment by davvd** [comment]
@g4s8 the last puzzle `516-ffc97ad1`/#534 originated from here solved

**Comment by 0pdd** [comment]
@g4s8 all 2 puzzles are solved here: [#725](https://github.com/yegor256/takes/issues/725), [#unknown]().
